homeassistant/components/freebox/const.py

```diff
@@ -48,3 +55,28 @@
     "vg_console": "mdi:gamepad-variant",
     "workstation": "mdi:desktop-tower-monitor",
 }
+
+ATTR_DETECTION = "detection"
+ATTR_ACTIVATION = "activation"
+ATTR_QUALITY = "quality"
+ATTR_SENSITIVITY = "sensitivity"
+ATTR_THRESHOLD = "threshold"
+ATTR_FLIP = "flip"
+ATTR_TIMESTAMP = "timestamp"
+ATTR_VOLUME = "volume"
+ATTR_SOUND_DETECTION = "sound_detection"
+ATTR_SOUND_TRIGGER = "sound_trigger"
+ATTR_RTSP = "rtsp"
+ATTR_DISK = "disk"
+
+CATEGORY_TO_MODEL = {
+    "pir": "F-HAPIR01A",
+    "camera": "F-HACAM01A",
+    "dws": "F-HADWS01A",
+    "kfb": "F-HAKFB01A",
+    "alarm": "F-MSEC07A",
+    "rts": "RTS",
+    "iohome": "IOHome",
+}
+
+HOME_COMPATIBLE_PLATFORMS = Platform.CAMERA
```

Comment:
```suggestion
HOME_COMPATIBLE_PLATFORMS = [
    Platform.CAMERA,
]
```
Don't know when that was reverted, but should be a list to work